### PR TITLE
Raspberry Pi Detector

### PR DESCRIPTION
**Pull request by PowerMacFan**
So I made this python script to try and detect a Raspberry Pi connected locally on your network. This is kinda useful when your trying to see if you can connect to your Raspberry Pi from your PC. It works by using the OS module to send a Ping Request to raspberrypi.local, The default local IP address for the Raspberry Pi. If the ping request goes through then a message will say there is a pi on your network, if it fails it will say that there is probably no pi on your network. 

Tl;Dr I made python script that scans for a Raspberry Pi and tells the user if there is one on their local network or not.

### CODE REVIEW

File: PyPrograms/Raspberry Pi Detector/main.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Raspberry Pi Detector
+# By @bupboi1337
+
+import os
+hostname = "raspberrypi.local" # Raspberry Pi Local Hostname
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/main.py]: I could have made hostname just host and make response just ping for a slightly smaller filesize

File: PyPrograms/Raspberry Pi Detector/main.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+response = os.system("ping -c 1 " + hostname) # Ping raspberrypi.local
+
+# check the response
+if response == 0: # If the ping went through correctly
+  print hostname, 'detected! There is a Raspberry Pi on your Network!'
+else: # If the ping was unsuccessful
+  print hostname, 'not found! There is probably not a Raspberry Pi on your network.'
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/main.py]: I should probably explain a bit more

File: PyPrograms/Raspberry Pi Detector/main.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Raspberry Pi Detector
+# By @bupboi1337
+
+import os
+hostname = "raspberrypi.local" # Raspberry Pi Local Hostname
+response = os.system("ping -c 1 " + hostname) # Ping raspberrypi.local
+
+# check the response
+if response == 0: # If the ping went through correctly
+  print hostname, 'detected! There is a Raspberry Pi on your Network!'
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/main.py]: I should replace detected! with connection successful and add a space before these messages so they don't look wierd

File: PyPrograms/Raspberry Pi Detector/readme.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+## Raspberry Pi Detector
+
+Scan for a Raspberry Pi on your network!
+Originally by @bupboi1337
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/readme.md:4]: Gotta add section for other people that edited this to add their names.

File: PyPrograms/Raspberry Pi Detector/main.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Raspberry Pi Detector
+# By @bupboi1337
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/main.py]: Instead of just "By" replace it with "Originally created by"

File: PyPrograms/Raspberry Pi Detector/requirements.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+os
```
Comment (PowerMacFan) [inline PyPrograms/Raspberry Pi Detector/requirements.txt:1]: Here's a joke for whoever is looking in here. 
Q: Whats a computers favorite game?

A: OSu!

**Review by PowerMacFan — COMMENTED** [review]
Its pretty good but I could have made this better when I made it.

**Comment by jayam04** [comment]
Sorry, busy today. its good one but can you change your program name to RaspberryPiDetector or raspberry-pi-detector or anything just shouldn't contain space and also change main.py with (newname).py , i wanted to talk about lisence before but want it private so made you contributar (maintainer) so we would talk in announcements tomorrow.

**Comment by jayam04** [comment]
Merge yourself once name is changed
